### ISSUE TITLE
Vulcanize runs on every client refresh

### ISSUE THREAD

**Issue by ThaumRystra** [Differential/meteor-vulcanize#27]
Vulcanize can take a substantial amount of time to run, and at the moment it doesn't seem to be caching its result, rather it runs on every client-side change.

**Comment by schnie** [comment]
Yes, this is a great idea.  Any ideas on how to implement?  I'll try to look into it, but will definitely accept a PR!

**Comment by ThaumRystra** [comment]
I'm still fairly clueless on Meteor's build plugin API, but they do give some guidance on [caching](https://github.com/meteor/meteor/wiki/Build-Plugins-API#caching). I'll also look into it; it should speed up client-side development quite a bit.
